ansible_base/rbac/api/serializers.py

```
@@ -26,6 +26,26 @@
+    When a content_type slug like 'justsomerandomstuff.organization' is provided
+    and not found, this checks whether the model portion exists under a valid service
+    and raises a specific error if the service name is unrecognized.
+    """
+
+    def to_internal_value(self, data):
+        try:
+            return super().to_internal_value(data)
+        except serializers.ValidationError:
+            # Check if the model portion exists under any valid service prefix
+            if isinstance(data, str) and '.' in data:
+                service_prefix, model_name = data.split('.', 1)
```

Ruff (0.16.3)

[warning] Unpacked variable `service_prefix` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)